### PR TITLE
OP-537 standardize table column headings

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-537.

### CODE REVIEW

File: src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java [hunk 1]
```diff
@@ -162,10 +162,10 @@
 			MessageBundle.getMessage("angal.common.name.txt").toUpperCase(),
 			MessageBundle.getMessage("angal.common.age.txt").toUpperCase(),
 			MessageBundle.getMessage("angal.common.sex.txt").toUpperCase(),
-			MessageBundle.getMessage("angal.admission.cityaddresstelephonenote.txt").toUpperCase(),
-			MessageBundle.getMessage("angal.admission.ward.txt").toUpperCase()
+			MessageBundle.getMessage("angal.admission.cityaddresstelephonenote.col").toUpperCase(),
+			MessageBundle.getMessage("angal.common.ward.txt").toUpperCase()
```
Comment (mwithi) [inline src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java:166]: in which case we should have .col or .txt?

**Review by mwithi — APPROVED** [review]
LGTM (just one question)

File: src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java:166]: I am using ".col" only when it is a table column heading; if it is both used as a column heading and it other places like dialog labels, or just text then I use ".txt".

**Review by mwithi — APPROVED** [review]
Approved! Thanks a lot!

**Pull request merged**